+neuron/hoc_pop.m

```diff
@@ -0,0 +1,13 @@
+function value = hoc_pop(returntype)
+% Pop an object of type returntype ("double", "string", "Object" or "void") off the NEURON stack.
+%   hoc_pop(returntype)
+    if (returntype=="double")
+        value = clib.neuron.hoc_xpop();
+    elseif (returntype=="string" || returntype=="char")
+        value = clib.neuron.matlab_hoc_strpop();
+    elseif (returntype=="Object")
+        value = clib.neuron.matlab_hoc_objpop();
+    elseif (returntype=="void")
+        value = 1.0;
```

Comment:
The returntype "void" is not used anywhere, I added it for testing purposes (just so that I did not have to pop anything off the stack, which can cause a crash if you are not careful). It is useful during development but not useful otherwise.

Reply:
Added a comment to the code for clarification.